sound/soc/sof/core.c

```diff
@@ -492,6 +489,9 @@ int snd_sof_device_remove(struct device *dev)
 	if (!IS_ERR_OR_NULL(pdata->pdev_mach))
 		platform_device_unregister(pdata->pdev_mach);
 
+	/* remove DSP */
```

Comment:
Please add some comments nearby to show the correct sequence of driver removing. You explanation in PR is very nice. Summarize and add to here will make sure others understand that.

Reply:
@xiulipan updated